README.md

```diff
@@ -55,6 +55,25 @@ This is the place to:
 Note: there is no need to repeat everything that is already well-documented in
 the code.
 
+If you would like to include math equations to explain parts of your code or repo,
+it is recommended to use an external renderer and insert prepared svg files here.
+For example, it's possible to insert a link to the online CodeCogs equation editor;
+below includes syntax to be read on darkmode by default in D-Wave's [Leap IDE](https://ide.dwavesys.io).
```

Comment:
I think it would more accessible to give two options:
* using online renderer
* Inserting an image file
And then explain in a footnote that the online renderer formulation has the ``filter:invert(1)`` part to look good in Leap's IDE, where darkmode is the default

Reply:
In the second point, it would also be good to recommend using a white background.

Reply:
Transparent background is actually preferable if the invert filter is used, since the IDE darkmode background is actually quite noticeably not jet black. It will not look absolutely disgusting though, so I think balancing what looks best with accessibility could to not express a preference on user image background.

Reply:
Perhaps it would be good to just note (since this is a guide) that the contributor should make sure that the equation is readable on both GitHub and the IDE.

Reply:
I'd word that as taking into consideration that some users such as IDE users might have dark backgrounds rather than phrasing it as a requirement that contributors ensure readability in multiple environments, that seems a burdensome requirement on 3rd-party contributors